File: test/fixtures/python/commented_json_with_session.py
from http.cookiejar import MozillaCookieJar
import requests

cookies = MozillaCookieJar('cookies.txt')

headers = {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
}

json_data = {}

with requests.Session() as session:
    session.cookies = cookies
    response = session.post('http://localhost:28139', headers=headers, json=json_data)
    # Note: json_data will not be serialized by requests
    # exactly as it was in the original request.
    #data = '{   }'
    #response = session.post('http://localhost:28139', headers=headers, data=data)
    cookies.save(ignore_discard=True, ignore_expires=True)

File: test/fixtures/swift/commented_json_with_session.swift
import Foundation

let jsonData = [:] as [String : Any]
let data = try! JSONSerialization.data(withJSONObject: jsonData, options: [])
// let data = "{   }".data(using: .utf8)

let url = URL(string: "http://localhost:28139")!
let headers = [
    "Content-Type": "application/json",
    "Accept": "application/json"
]

var request = URLRequest(url: url)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers
request.httpBody = data as Data

let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print(error)
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print(str ?? "")
    }
}

task.resume()
